### ISSUE TITLE
enum fields (auto generated) are inconsistent

### ISSUE THREAD

**Issue by lost22git** [protostuff/protostuff-compiler#77]
message.proto
```proto
  enum EncodingType {
    kEncodingNone = 0;
    kEncodingLz4 = 1;
  }
```
java object (generated by compiler)

![image](https://user-images.githubusercontent.com/65008815/166161736-e57f7ab7-d5d1-44cf-b9be-0df2bc496322.png)

![image](https://user-images.githubusercontent.com/65008815/166161787-bf334135-49ab-4a7f-854c-d2a94f279774.png)

**Comment by lost22git** [comment]
The problem is due to v3.1.38

![image](https://user-images.githubusercontent.com/65008815/166171162-cd6ca953-6a1c-4a2e-b4a3-fda3263fd0a4.png)

but it is resolved in master branch

![image](https://user-images.githubusercontent.com/65008815/166171370-b4d57649-0ca3-47cd-9a12-a813a67717d2.png)

so please publish the latest package to maven repo, thanks

![image](https://user-images.githubusercontent.com/65008815/166171604-bf134c6a-5e95-4996-86c0-5196a65adbbd.png)

**Comment by kshchepanovskyi** [comment]
I'll check and publish new release, thanks!

**Comment by kshchepanovskyi** [comment]
Published version 3.1.40
https://repo1.maven.org/maven2/io/protostuff/protostuff-parser/3.1.40/
